src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -88,5 +91,38 @@ def generate_strategy_pool():
     pool.add_strategy(strategy.FORK_STRATEGY)
   if do_strategy(strategy.MUTATOR_PLUGIN_STRATEGY):
     pool.add_strategy(strategy.MUTATOR_PLUGIN_STRATEGY)
+  return pool
+
+
+def generate_strategy_pool():
+  """Generate a strategy pool based on probability
+  distribution from multi armed bandit experimentation."""
+  query = data_types.FuzzStrategyProbability.query()
+  distribution = list(ndb_utils.get_all_from_query(query))
+
+  # If we are not able to query properly,
+  # draw randomly according to probability parameters.
+  if not distribution:
+    return generate_random_strategy_pool()
+
+  strategy_selection = utils.random_weighted_choice(distribution, 'probability')
+  strategy_name = strategy_selection.strategy_name
 
+  strategies = strategy_name.split(',')
+  pool = StrategyPool()
+
+  if 'radamsa' in strategies:
```

Comment:
Maintaining yet another list of strings for each strategy seems problematic. Could you use the full strategy names in the query that stores these? If we also maintained a list of all possible strategies for libFuzzer here we could use it to detect invalid strategies (to catch the issue we discussed today), and also simplify some of this code by just iterating through the list, checking for the strategy name, and adding it to the pool if it's included.